accounts/models.py

```diff
@@ -137,20 +139,25 @@ def email_user(self, subject, message, from_email=None, **kwargs):
             subject, message, from_email, [self.email], **kwargs)
 
 
-class AuditLogEvent(django.db.models.Model):
+class AbstractAuditLogEventBase(django.db.models.Model):
     created_on = django.db.models.DateTimeField(auto_now_add=True)
     updated_on = django.db.models.DateTimeField(auto_now=True)
     recorded_on = django.db.models.DateTimeField(auto_now_add=True)
 
     user_id = django.db.models.IntegerField(_('User ID'), db_index=True)
     user_email = django.db.models.EmailField(_('User Email'), db_index=True)
-    company = django.db.models.ForeignKey('accounts.Company')
+    company = django.db.models.ForeignKey('accounts.Company', null=True, on_delete=django.db.models.SET_NULL,
+                                          related_name='%(app_label)s_%(class)s_users')
```

Comment:
@mohannadsh related_name should reference to the current model, I think it should be something like `%(app_label)s_%(class)s_logs`